refs/src/Effect/Ref.ss

```diff
@@ -6,38 +6,41 @@
           read
           modifyImpl
           write)
-  (import (prefix (chezscheme) scm:))
+  (import (prefix (chezscheme) scm:)
+          (prefix (purs runtime) rt:))
 
   (scm:define _new
     (scm:lambda (v)
       (scm:lambda ()
-        (scm:box v))))
+        (scm:list (scm:box v) (scm:make-mutex)))))
```

Comment:
I think you could use a pair (`cons`) for this, then accessing the mutex would be slightly easier.

Reply:
Ah yes good point